content/en/docs/pipelines/sdk/pipelines-metrics.md

```diff
@@ -40,7 +40,7 @@ See the
 The metrics file has the following requirements:
 
 * The file path must be `/mlpipeline-metrics.json`.
-* `name` must follow the pattern `^[a-z]([-a-z0-9]{0,62}[a-z0-9])?$`.
+* `name` must follow the pattern `^[a-zA-Z]([-_a-zA-Z0-9]{0,62}[a-zA-Z0-9])?$`. (Until 0.5.2, `^[a-z]([-a-z0-9]{0,62}[a-z0-9])?$` must be followed).
```

Comment:
Suggested revision:

* `name` must match the following pattern.
  `^[a-zA-Z]([-_a-zA-Z0-9]{0,62}[a-zA-Z0-9])?$`
  
  For Kubeflow Pipelines versions before 0.5.2, `name` must match the following pattern.
  `^[a-z]([-a-z0-9]{0,62}[a-z0-9])?$`

Reply:
Thanks! will udpate